### DISCUSSION TITLE
Rest Framework for use with Next API Routes?

### DISCUSSION THREAD

**Discussion by blainegarrett** [vercel/next.js#13099] [Help]
Greetings. I'm porting an application that has a Python 2.7 back standalone REST service. It only exists to service the frontend (which has now been fully ported to Next.js). I'm starting to look into porting the REST service. I've leveraged Next.js api routes for a few things and it seems promising for my needs. However, I really like REST frameworks like FastAPI (python 3). Is there a powerful REST api framework for Node (preferably typescript ) that plugs in easily with Next.js API routes? I've looked a bit at NEST.JS but couldn't find anyone using next and nest together easily.

**Comment by w3bdesign** [comment]
Have you looked at https://feathersjs.com/ ?

**Reply by blainegarrett** [reply]
I will check it out. Thanks. Does this work well with next?
